src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveLevelService.kt

```diff
@@ -26,7 +26,7 @@ import uk.gov.justice.digital.hmpps.incentivesapi.dto.IncentiveLevelUpdate as In
 class IncentiveLevelService(
   private val clock: Clock,
   private val incentiveLevelRepository: IncentiveLevelRepository,
-  private val prisonIncentiveLevelService: PrisonIncentiveLevelService,
+  private val prisonIncentiveLevelService: PrisonIncentiveLevelService, // OK to inject unaudited version as audited subclass passes in audited form instead
```

Comment:
Seems like rule of thumb is "always use the audited version" - could this service also use the `PrisonIncentiveLevelAuditedService` one too?

Reply:
the subclass does this:
https://github.com/ministryofjustice/hmpps-incentives-api/blob/25147e7aa62c25cedcabfe6c2a4130fa9a4839db/src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveLevelAuditedService.kt#L25

_this_ class (`IncentiveLevelService`) is unaudited so does not need to require _only_ the audited interface of a peer dependency.

Reply:
I suppose I was thinking of the two services as separate (incentive level vs prison incentive level one) but I guess makes sense to use the unaudited prison level service in this context of unaudited incentive level service.

Reply:
Also, good point about the "does not need to require _only_ the audited" one - I forgot you wired things in such a way you could use the audited version when a simple version is required (in other words an instance of `PrisonIncentiveLevelService` is-a `PrisonIncentiveLevelService`)